src/main/java/org/kitteh/irc/client/library/implementation/ManagerISupport.java

```diff
@@ -307,6 +307,49 @@ private ISupportTopicLen(@Nonnull Client client, @Nonnull String name, @Nonnull
         }
     }
 
+    private static final class ISupportExtBan extends IRCISupportParameter implements ISupportParameter.ExtBan {
+        private final Optional<Character> prefix;
+        private final List<ChannelMode> extBans;
+
+        private ISupportExtBan(@Nonnull Client client, @Nonnull String name, @Nonnull Optional<String> value) {
+            super(client, name, value);
+            if (!value.isPresent()) {
+                throw new KittehServerISupportException(name, "No extbans defined");
+            }
+
+            final String[] split = value.get().split(",");
```

Comment:
For mental parsing, a comment above this posting the full line of ISUPPORT EXTBAN would be appreciated.